news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,239 @@
+
+Try this map and spice up your collection!
+
+*written by [KnightC0re](https://osu.ppy.sh/users/7894340)*
+
+## <a id="taiko"></a> osu!taiko
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/leviathan.jpg)](https://osu.ppy.sh/beatmapsets/1013884)
+
+Topping off this season's Spotlights, we have a map that keeps you busy with complicated rhythms all the way through - Leviathan, a mostly piano-based Touhou arrangement, mapped by [yassu-](https://osu.ppy.sh/users/7095592), a mapper who's shown their skill at handling rhythmically complicated songs time and time again! 
+
+In this map specifically, a lot of the complexity comes from blue tick rhythms and tricky off-hand patterns mixed with 1/6 and 1/8 for even more handswitches whenever the piano calls for it, as well as some standalone 1/8 doubles sneaked in which feel really pleasant to play - all in all a great fit for spotlights, even more so as the set as a whole manages to maintain its rhythmical complexity across the lower difficulties, making it a treat for people at all levels seeking a challenge!
+
+*written by [Greenshell](https://osu.ppy.sh/users/8693851)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/dungeon.jpg)](https://osu.ppy.sh/beatmapsets/1056653)
+
+On second place we have sakuraburst - dungeon music by [IceOC](https://osu.ppy.sh/users/5482401). Outstanding SV map that can be a tough challenge for any player despite its simplicity and low star rating. 
+
+IceOC shows how much details this map has with variations from x0.13 to x10.00 slider velocity, applied on high-speed ninja finishers, extremely slow sections and tricky patterns, everything attached to an exceptional representation of music in an entertaining and interesting way. Such uniqueness and clean execution make this a well-deserved spotlight.
+
+*written by [Ulqui](https://osu.ppy.sh/users/1263669)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/1000.jpg)](https://osu.ppy.sh/beatmapsets/1032172)
+
+Coming in third this season, we have the creatively named 1000 gecs by the artist 100 gecs. 
+
+Please beware, this is an extremely long map by [Genjuro](https://osu.ppy.sh/users/3196091), sporting 23 minutes of silly songs backed by a solid and diverse map! If you have half an hour to spare, it will not be wasted on playing this map, which will surely put a smile on your face not only due to the song choice, but also due to how they are mapped. This map is clearly a work of passion from Genjuro!
```

Comment:
```suggestion
Please beware - this is an extremely long map, sporting 23 minutes of silly songs backed by a solid and diverse map! If you have half an hour to spare, it will not be wasted on playing this map, which will surely put a smile on your face not only due to the song choice, but also due to how they are mapped. This map is clearly a work of passion from Genjuro!
```